osquery/extensions/impl_thrift.cpp

```diff
@@ -343,6 +344,17 @@ void ExtensionRunnerInterface::init(RouteUUID uuid, bool manager) {
     server_->processor =
         std::make_shared<extensions::ExtensionManagerProcessor>(handler);
   }
+  // Set the global output function for thrift
+  GlobalOutput.setOutputFunction([](const char* message) -> void {
+    time_t now;
```

Comment:
I think we can skip the `dbgtime` output since GLOG will insert a timestamp itself. I am not aware of a reason we'd want both?

Reply:
Thanks, @theopolis. I removed the time wrapper from the error log. I was not sure of its use with GLOG and wanted your feedback.